libs/island-ui/core/src/lib/Tabs/Tabs.stories.tsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react'
+import { withDesign } from 'storybook-addon-designs'
 import { Tabs } from './Tabs'
 import { Stack } from '../Stack/Stack'
 import { Typography } from '../Typography/Typography'
 import { ContentBlock } from '../ContentBlock'
 
+const figmaLink =
+  'https://www.figma.com/file/pDczqgdlWxgn3YugWZfe1v/UI-Library-%E2%80%93-%F0%9F%96%A5%EF%B8%8F-Desktop?node-id=208%3A697'
```

Comment:
Shouldn't this be a config rather?

Reply:
I started with a config like so:

```
import { figmaLinks } from '../../../storybook'
const figmaLink = figmaLinks['FormStepper']
```
but changed my mind 🤷 not sure where to put the config.

Reply:
You could create an `environment.ts` file where you define the Figma links array

Reply:
I don't think this warrants an environment file

Reply:
I personally do not think this is a problem unless the designers will change the URLs rapidly. If this turns ugly at some point, we should consider moving these links to a config where it could be simpler to update the links without making a PR going through the CD pipeline.